bundles/org.openhab.binding.bloomsky/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,547 @@
+	</thing-type>
+
+	<channel-group-type id="storm-observations">
+		<label>Storm Device Weather Observations</label>
+		<description>This is the weather sensor observations from the Storm device</description>
+		<channels>
+			<channel typeId="stormUVIndex" id="stormUVIndex"></channel>
+			<channel typeId="system.wind-direction" id="windDirectionAngle"></channel>
+			<channel typeId="windDirectionCompass" id="windDirectionCompass"></channel>
+			<channel typeId="rainDaily" id="rainDaily"></channel>
+			<channel typeId="windGust" id="windGust"></channel>
+			<channel typeId="windChill" id="windChill"></channel>
+			<channel typeId="sustainedWindSpeed" id="sustainedWindSpeed"></channel>
+			<channel typeId="rainRate" id="rainRate"></channel>
+			<channel typeId="rain24h" id="rain24h"></channel>
+			<channel typeId="stormTimeStamp" id="stormTimeStamp"></channel>
+		</channels>
+	</channel-group-type>
+
+	<!-- Storm Specific Channel Types -->
+
+	<channel-type id="stormUVIndex">
+		<item-type>Number</item-type>
+		<label>UV Index</label>
+		<description>UV index</description>
+		<state readOnly="true" pattern="%d"></state>
+	</channel-type>
+
+	<channel-type id="rainRate">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation Rate</label>
+		<description>Precipitation rate</description>
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%/hr"/>
+	</channel-type>
+
+	<channel-type id="sustainedWindSpeed">
+		<item-type>Number:Speed</item-type>
+		<label>Sustained Wind Speed</label>
+		<description>Sustained Wind Speed</description>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="rain24h">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation 24h</label>
+		<description>Precipitation last 24 hours</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="rainDaily">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation Total Past 24h</label>
+		<description>Precipitation total last 24 hours</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="windDirectionCompass">
+		<item-type>String</item-type>
+		<label>Wind Direction Compass</label>
+		<description>Wind direction as compass point (N, NE, E, SE, S, SW, W, NW)</description>
+		<state readOnly="true" pattern="%s"></state>
+	</channel-type>
+
+
+	<channel-type id="windGust">
+		<item-type>Number:Speed</item-type>
+		<label>Wind Gust</label>
+		<description>Wind gust</description>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="windChill">
+		<item-type>Number:Temperature</item-type>
+		<label>Wind Chill</label>
+		<description>Wind chill in °F or °C</description>
```

Comment:
remove any referrence to C or F as it is automatic due to your using UOM.

Reply:
Removed temp indicator as suggested.